data/json/recipes/food/other.json

```diff
@@ -50,6 +50,52 @@
     "tools": [ [ [ "surface_heat", 10, "LIST" ] ] ],
     "components": [ [ [ "water", 1 ], [ "water_clean", 1 ] ], [ [ "honey_bottled", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "young_koji",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "skill_used": "cooking",
+    "difficulty": 5,
+    "skills_required": [ "survival", 6 ],
+    "time": "2 m",
+    "proficiencies": [ { "proficiency": "prof_food_prep" }, { "proficiency": "prof_fermenting" } ],
+    "book_learn": [ [ "brewing_cookbook", 6 ] ],
+    "components": [ [ [ "rice_cooked", 1 ] ], [ [ "ash", 1 ] ], [ [ "empty_corn_cob", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "NO_EXERCISE",
+    "result": "soy_wheat_dough",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "skill_used": "cooking",
+    "difficulty": 5,
+    "time": "4 h",
+    "proficiencies": [ { "proficiency": "prof_food_prep" }, { "proficiency": "prof_fermenting" } ],
+    "book_learn": [ [ "cookbook_sushi", 6 ] ],
+    "qualities": [ { "id": "COOK", "level": 3 } ],
+    "tools": [ [ [ "surface_heat", 80, "LIST" ] ], [ [ "mortar_pestle", -1 ], [ "food_processor", 20 ] ] ],
+    "components": [ [ [ "soybean", 1 ] ], [ [ "flour", 10 ] ], [ [ "water", 4 ], [ "water_clean", 4 ] ], [ [ "paper", 1 ] ] ]
```

Comment:
Would it be possible to be able to use another wheat-free flour here?

Reply:
Not really; most soy sauces specifically are made with wheat flour in addition to soybeans. Tamari soy sauce is made with just soybeans and there are some gluten-free soy sauce substitutions made in a completely different way, but no recipes seem to use non-wheat flour.